script/integration/containerd/entrypoint.sh

```diff
@@ -247,4 +250,8 @@ diff --no-dereference -qr "${USR_NORMALSN_PLAIN_STARGZ}/" "${USR_STARGZSN_PLAIN_
 ctr-remote --namespace=dummy images rpull --user "${DUMMYUSER}:${DUMMYPASS}" \
            "${REGISTRY_HOST}:5000/ubuntu:esgz"
 
+############
+# Test for starting when no configuration file.
+reboot_containerd nonexistent.toml
```

Comment:
When a non-existent path was explicitly specified, it should fail

Reply:
I misunderstood.
https://github.com/containerd/stargz-snapshotter/pull/229/commits/7d99fc8b3efc038f7bb27e031f601230caecac98